### ISSUE TITLE
MPOS DB

### ISSUE THREAD

**Issue by docskyhawk** [bonesoul/CoiniumServ#911]
I have installed coiniumserv on windows server 2016 as well as mysql.  I created the database/user and enabled MPOS in the config.  I have a coin daemon running and added it to a new pool.  Everything appears to work correctly except a couple of things.  The console on coinium shows it connects to mysql as expected but the database still doesn't have any tables or records.  When I try to connect with a miner, the minor kicks an unauthorized worker error.  If I disable MPOS, the miners can connect and starts hashing fine.  Does something else need installed for MPOS?  Is there a script to create the tables?  

The second issue is that the console keeps showing no blocks found and according to the web UI, there are workers but not showing any hash rates or activity.  Is this due to no storage used while MPOS is disabled?  

Update: I found the SQL file to create the tables and will give that a go.  What is normal to see in the console output when there are miners working?  The only thing so far has been the occasional no blocks found message?

**Comment by bonesoul** [comment]
sounds like a hashrate calculation error.

**Comment by docskyhawk** [comment]
Update:  I was able to import the tables from MPOS.  

Now, when I try to connect to the pool using cgminer using a wallet address, it kicks an unauthorized worker error. (Wallet address edited)  This only happens if I enable MPOS.  If I disable storage, it works "fine".

 Miner authentication failed: SdfsdfdSGFyZ3Xsdfbq38VXYo1fHJJ

I inserted a row into pool_worker with the wallet address, password, etc and coiniumserv shows it connected but the miner kicks a pool 0 JSON stratum auth failed: [
"data": {
      "Source": "CoiniumServ",
      "RemoteStackIndex": 0,
      "InnerException": null,
      "StackTraceString": "   at CoiniumServ.Persistance.Layers.Mpos.MposStorage.AddAccount(IAccount account) in C:\\
